conjur-openapi.yml

```diff
@@ -25,6 +25,8 @@ tags:
   description: "SSH keys"
 - name: "resources"
   description: "Resources"
+- name: "whoami"
+  description: "WhoAmI"
```

Comment:
does this one endpoint merit its own section in the spec? maybe we should have a `status` section that includes the server status and whoami?

an argument could be made that authenticator status (and maybe other future config-relevant routes we add) should also be in this section. cc @alexkalish

Reply:
I agree, this would make a lot more sense based on what the endpoints actually do. I think possibly the /authenticators endpoint would also be included in this tag as well.